tests/test_web.py

```diff
@@ -0,0 +1,115 @@
+# clear-dissector-web - tests for clear dissector web application setup
+# by docker and docker compose
+#
+# Copyright (C) 2019 Intel Corporation
+#
+# Licensed under the MIT license, see COPYING.MIT for details
+# NOTE: requires pytest-selenium plugin.
+# Run : eg. pytest --driver Chrome
+# Run : eg. pytest --driver Chrome --clr-url <custom_clear_app_url>
+# where default clr-url=http://localhost:8080
+
+import os
+import subprocess
+import pytest
+
+basepath = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+username = 'testuser'
+password = 'changeme1234!'

```

Comment:
It's not a recommended approach to hardcode any sensitive information regarding username/password, hostname, IP, etc.

Reply:
Thanks for the feedback, we will look into this.